### PR TITLE
x86: base-files add support for Sophos 135r3/135r3w

### PR DESCRIPTION
**Pull request by stangri**
The Sophos SG/XG-135 revision 3 has odd numbering of eth ports where the WAN port (as marked on the case) is `eth6` and `eth0`, `eth1`, `eth2`, `eth3`, `eth5`, `eth7`, `eth8` are LAN ports. 

Port `eth4` seems to be reserved for the expansion card.

Original discussion at https://forum.openwrt.org/t/openwrt-on-revision-3-of-sophos-desktop-appliances/152912

If possible, please cherry-pick for OpenWrt 22.03.

Signed-off-by: Stan Grishin <stangri@melmac.ca>

### CODE REVIEW

**Comment by stangri** [comment]
@chunkeey You have merged my previous PR for older devices, can you please merge this? I'm struggling to undo the merge commit, let me know if I should create another PR.

**Comment by stangri** [comment]
Thanks @hauke! Can I convert this to WIP or should I close it if it's not ready? The 'eth4' port is probably SFP and if it's the case does it need to be defined in the product-specific dts file?

**Comment by hauke** [comment]
Please do not push merge commits. Do a rebase instead of you want to change your pull request. 

Does the SFP port work for you?
This is an x96 device it probably uses ACPI instead of device tree.

**Comment by stangri** [comment]
> Please do not push merge commits. Do a rebase instead of you want to change your pull request.
Sorry about that, I was trying to push a change to the 01_leds file.

> Does the SFP port work for you? This is an x96 device it probably uses ACPI instead of device tree.
I don't know how to test the SFP port, never had one on a prior hardware, I'd appreciate some guidance. I've installed the `kmod-sfp` but it doesn't list as anything was actually installed:
```sh
XG-135r3 in ~ # opkg files kmod-sfp
Package kmod-sfp (5.10.161-1) is installed on root and has the following files:
XG-135r3 in ~ #
```

Also, I've added the single LED which is reported in the `/sys/class/leds/` to `target/linux/x86/base-files/etc/board.d/01_leds` and I've tried to follow the wiki on adding the new device but there's no `/sys/class/gpio/gpiochip*/` nor `/sys/kernel/debug/gpio`.

What can I read to familiarize myself with the process of adding a new x86 device/sfp device? I've tried to browse for prior commits but didn't find anything useful for x86.

**Comment by aparcar** [comment]
Looks good to me

**Comment by stangri** [comment]
I'm struggling with needing to add another file and the git-fu/rebase. :( 

I may have to close this and create another PR as I'm not sure how to proceed with this one.

**Comment by aparcar** [comment]
So sweat I'll take care of it

**Comment by stangri** [comment]
@aparcar Thanks Paul! I've been trying to add two files in `target/linux/x86/base-files/etc/board.d/`.

If you can refer me to something I can read about adding SFP support/port, I'd be grateful.

**Comment by stangri** [comment]
@aparcar Paul, sorry I'm not sure if this has been merged, I could only find this commit by @chunkeey : https://git.openwrt.org/?p=openwrt/openwrt.git;a=commit;h=ffab23d99d980974e502989994f3aaec3f462865 when searching for XG-135.

Please close if this has been merged or let me know if I can do something to have it merged.

File: target/linux/x86/base-files/etc/board.d/01_leds [hunk 1]
```diff
@@ -16,6 +16,13 @@ pc-engines-apu1|pc-engines-apu2|pc-engines-apu3|pc-engines-apu4|pc-engines-apu5|
 	ucidef_set_led_netdev "lan" "LAN" "apu:green:2" "br-lan"
 	ucidef_set_led_default "diag" "DIAG" "apu:green:1" "1"
 	;;
+sophos-sg-105wr1|sophos-sg-125wr1|sophos-sg-135wr1|sophos-xg-105wr1|sophos-xg-125wr1|sophos-xg-135wr1)
+	ucidef_set_led_netdev "wlan" "WiFi" "ath9k-phy0" "phy0tpt"
+	;;
+sophos-sg-105wr2|sophos-sg-125wr2|sophos-sg-135wr2|sophos-xg-105wr2|sophos-xg-125wr2|sophos-xg-135wr2|
```
Comment (aparcar) [inline target/linux/x86/base-files/etc/board.d/01_leds:22]: Missing `\`?

**Comment by aparcar** [comment]
> If you can refer me to something I can read about adding SFP support/port, I'd be grateful.

Sorry I can't, never touched that stuff.

> Please close if this has been merged or let me know if I can do something to have it merged.

The first three were merged but not your additional commit.

File: target/linux/x86/base-files/etc/board.d/01_leds (hunk 1 above)
Comment (stangri) [reply target/linux/x86/base-files/etc/board.d/01_leds]: Thanks, I've tried to fix it, but not sure why bcm47xx files were added to this commit.

**Comment by chunkeey** [comment]
Merged the x86 parts. Thanks.

**Pull request closed without merging**